### ISSUE TITLE
Unable to download OFI resources using 'Access/Download' button from resource page url

### ISSUE THREAD

**Issue by Mbrownshoes** [bcgov/ckanext-bcgov#281]
When attempting to download an OFI resource using the clickable URL, often the modal window fails to load and the spinner is shown:
![screen shot 2017-01-24 at 1 51 09 pm](https://cloud.githubusercontent.com/assets/3362533/22268367/2968243e-e23c-11e6-9bb4-878eb73a80af.png)
with the following console errors
![screen shot 2017-01-24 at 1 27 45 pm](https://cloud.githubusercontent.com/assets/3362533/22268393/409da8f4-e23c-11e6-8418-0c50806800a1.png)

@ll911 This does not occur when the Access/Download button is clicked, only the URL
![screen shot 2017-01-24 at 3 34 51 pm](https://cloud.githubusercontent.com/assets/3362533/22271684/bc6e7c48-e24a-11e6-9e17-3fee9adf229c.png)

**Comment by Mbrownshoes** [comment]
@jrods after further exploration, we do occasionally see the spinner with both the 'Access/Download' button and the URL when on the resource page. 
It appears as though the url is different when downloading from the main record page:
![screen shot 2017-01-24 at 4 01 16 pm](https://cloud.githubusercontent.com/assets/3362533/22272372/8b530620-e24e-11e6-9c87-0e8326a3b488.png)
URL for this is https://cat.data.gov.bc.ca/dataset/pub-downloadofi#edc-mow

while the url at the resource page is:
https://cat.data.gov.bc.ca/dataset/pub-downloadofi/resource/3bb5c689-27fd-4550-8016-8c75701fba26#edc-mow

**Comment by Mbrownshoes** [comment]
<img width="1098" alt="screen-shot-2017-01-25-at-12 32 30-pm" src="https://cloud.githubusercontent.com/assets/3362533/22309209/264d6dcc-e2ff-11e6-99c5-bd10a60f83cc.png">
<img width="1102" alt="screen-shot-2017-01-25-at-12 32 41-pm" src="https://cloud.githubusercontent.com/assets/3362533/22309210/265fa910-e2ff-11e6-9bfd-8947c3989dc7.png">

**Comment by ll911** [comment]
workaround is update resource.config
```
cat resource.config
# http://docs.ckan.org/en/latest/contributing/frontend/resources.html

[main]
dont_bundle = jquery-ui.js jquery.fileupload.js jquery.iframe-transport.js image_upload.js responsive.js search.js typehead.bundle.js ofi/modal.js

order = jquery-ui.js jquery.iframe-transport.js jquery.fileupload.js image_upload.js responsive.js

[IE conditional]

lte IE 8 = edc_theme_ie8.css
```

**Comment by Mbrownshoes** [comment]
@kfishwick @jrods

**Comment by Mbrownshoes** [comment]
@jrods @kfishwick can you please review @ll911's proposed fix above and incorporate into the repo. I will create a new ticket for the next release (1.7) for you to make sure that the js libraries do work when bundled.

**Comment by jrods** [comment]
@Mbrownshoes i can't reproduce the issue you are having on cat, i've tried both my idir and the dbcuser2 account. I don't believe it's the bundle that's causing issues, but I will have a look.

**Comment by ll911** [comment]
@jrods cat is patched at runtime with my changes, so you can't reproduce the issue. you can try locally and serve those js in bundled and will get the error.

**Comment by jrods** [comment]
@ll911 are these changes on cad right now?

**Comment by ll911** [comment]
@jrods nope, they are not in cad, but they are not bundled when serving in cad, not sure if that because of hardware restriction on cad instance for single worker + single thread.

**Comment by jrods** [comment]
@ll911 `ofi/modal.js` is incorrect, it should be `ofi/ofi_modal.js`

**Comment by ll911** [comment]
@jrods actually only file should be exempt from bundle is ```typeahead.bundle.js```

**Comment by Mbrownshoes** [comment]
Leo implemented a fix in 1.6.2. In cat now.
